packages/plugin-bundle-webpack/plugin.js

```diff
@@ -109,14 +116,16 @@ module.exports = function plugin(config, args) {
         ],
       };
 
-      const stats = await compilePromise({
-        ...webpackConfig,
-        entry: path.join(srcDirectory, entryPoint),
-        output: {
-          path: destDirectory,
-          filename: "js/bundle-[hash].js",
-        },
-      }).catch((err) => {
+      const stats = await compilePromise(
+        extendConfig({
+          ...webpackConfig,
+          entry: path.join(srcDirectory, entryPoint),
+          output: {
```

Comment:
It would be great to see a logged warning if the user attempted to define one of these themselves (`if (webpackConfig.entry) console.error(...)`), since they'll be confused why it didn't work

Reply:
Right now they ARE able to overwrite both the entry and output. I wasn't sure if I should do this:
```
extendConfig({
  ...webpackConfig,
  entry: path.join(srcDirectory, entryPoint),
  output: {
    path: destDirectory,
    filename: "js/bundle-[hash].js",
  },
})
```
OR this:
```
{
  ...extendConfig(webpackConfig),
  entry: path.join(srcDirectory, entryPoint),
  output: {
    path: destDirectory,
    filename: "js/bundle-[hash].js",
  },
}
```

But I decided on the first option (which allows them override entry and output). I figure if the users want to mess with the entry and output we should let them. Anyone using this option hopefully knows what they're doing.

What do you think?

Reply:
Oh nice, gotcha. Yup, I think that makes sense. We could theoretically still add a warning if the user tries to set that, but I guess you're right that they'd really be intentionally messing with something that they shouldn't. I think that's fine to leave for later, if we get reports of confused users.